File: JUNE20B/PRICECON.cpp
//Link to solution https://www.codechef.com/viewsolution/33645298

#include<bits/stdc++.h>
using namespace std;
int main(){
  int t,i;
  cin>>t;
  for(i=0;i<t;i++){
    int n,k,j;
    cin>>n>>k;
    int arr[n];
    for(j=0;j<n;j++){
      cin>>arr[j];
    }
    int sum=0,sum1=0;
    for(j=0;j<n;j++){
      sum+=arr[j];
    }
    for(j=0;j<n;j++){
      if(arr[j]>k){
        arr[j]=k;
      }
    }
    for(j=0;j<n;j++){
      sum1+=arr[j];
    }
    cout<<(sum-sum1)<<"\n";
  }
  return 0;
}

File: JUN20B/PRICECON.java
//link to my solution-https://www.codechef.com/viewsolution/38403554

import java.util.*;
class A
{
    public static void main (String[] args)
    {
        Scanner sc=new Scanner(System.in);
        int T=0,n=0,p=0,k=0,i=0,j=0,lr=0;
        T=sc.nextInt();
        for (j=0;j<T ;j++ )
        {
            n=sc.nextInt();
            k=sc.nextInt();
            for(i=0;i<n;i++)
            {
                p=sc.nextInt();
                if(p>k)
                {
                    lr=lr+p-k;
                 }
            }
            System.out.println(lr);
            lr=0;
            
        }
        
    }
}